hi-ml-cpath/src/SSL/configs/HistoSimCLRContainer.py

```diff
@@ -49,16 +49,21 @@ def _get_transforms(self, augmentation_config: Optional[CfgNode],
                 val_transforms = DualViewTransformWrapper(val_transforms)  # type: ignore
         return train_transforms, val_transforms
 
-    @staticmethod
-    def get_transforms(apply_augmentations: bool) -> ImageTransformationPipeline:
-        transforms: List[Any] = []
+    def get_preprocessing_transforms(self) -> List[Any]:
+        return([])
+
+    def get_augmentations(self) -> List[Any]:
```

Comment:
[Minor] Can you please change Any by Callable?

Reply:
sure, done